src/main/types/SIOP.types.ts

```diff
@@ -416,6 +416,11 @@ export enum ResponseContext {
   OP = 'op',
 }
 
+export enum LinkedDomainValidationMode {
+  NEVER = 'never', // We don't want to verify Linked domains
+  OPTIONAL = 'optional', // If present, did-auth-siop will check the linked domain
+  ALWAYS = 'always', // We'll always check the linked domains
```

Comment:
Probably require is a better name. And we should make clear in the comment that this will result in an error if no linked domain is present

Reply:
I'm not sure if you're talking about the name of the enum or enum values. Maybe change the name to `CheckDidLinkedDomain` and for the values:
- `NEVER`
- `CHECK_IF-AVAILABLE`
- `ALWAYS_CHECK`

Reply:
changed to:
```
NEVER
IF_PRESENT
ALWAYS
```